claripy/ast/strings.py

```diff
@@ -151,6 +154,7 @@ def StringV(value, length=None, **kwargs):
 StrToInt = operations.op("StrToInt", (String, int), BV, calc_length=operations.strtoint_bv_size_calc, bound=False)
 IntToStr = operations.op("IntToStr", BV, String, calc_length=operations.int_to_str_length_calc, bound=False)
 StrIsDigit = operations.op("StrIsDigit", String, Bool, bound=False)
+UnitStr = operations.op("UnitStr", BV, String, bound=False) # convert BV to single-char string
```

Comment:
The meaning of having a single type as opposed to a tuple of types in the input argument is that it accepts a variadic number of arguments of that type. Is that what you want here? The other operations may be specified wrong... I don't know their semantics off the top of my head.

Reply:
OK, I will need to fix a few of the implementations (which are not variadic and should be) and fix my signature for `UnitStr`. I'll make a commit with those changes tomorrow.